README.md

```diff
@@ -1 +1,97 @@
-# weatherapp
\ No newline at end of file
+# Weather App
+
+ This weather App is created to show a list of weather informations for different cities in Nigeria.
+
+## Built With
+
+* HTML
+* CSS
+* JS
+
+## Framework
+
+* React
+
+## Getting Started
+
+Tools Needed
+
+* Text editor
+* Node.js
+* Gitflow
+
+Clone this repository on to your local PC
+
+* Click on the Code green button
+* By the right end of the read-only input containing the repository link click the clipboard icon to copy the link
+* In your local PC, open your terminal in the folder you would like to clone the repository into
+* Clone the repository with the command: git clone (copied link); like so: <https://github.com/OyePriscilla/weatherapp>
+* After the clone, type in the command cd weatherapp to access the directory on the terminal.
+
+* Once the files are on your machine, open the weatherapp folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
```

Comment:
You didn't add these commands like `npm install` which is gonna install the dependencies, `npm start` which is gonna start the application and `npm test` which is gonna run the tests. :sob: Please add these commands in your getting started section because you wanna tell other people how to start your application properly :+1: